src/examples.jl

```diff
@@ -169,7 +169,7 @@ const _examples = PlotExample[
         )],
     ),
     PlotExample( # 9
-        "",
+        "Build plot in pieces (continued)",
```

Comment:
Those headers will make this extra cards, I don't think there is much benefit to that

Reply:
I didnt think of that, you're right.